lnbits/wallets/spark.py

```diff
@@ -199,9 +199,7 @@ async def get_payment_status(self, checking_id: str) -> PaymentStatus:
         if r["pays"][0]["payment_hash"] == checking_id:
             status = r["pays"][0]["status"]
             if status == "complete":
-                fee_msat = -int(
-                    r["pays"][0]["amount_sent_msat"] - r["pays"][0]["amount_msat"]
-                )
+                fee_msat = -(int(r["pays"][0]["amount_sent_msat"][0:-4]) - int(r["pays"][0]["amount_msat"][0:-4]))
```

Comment:
I suppose the `[0:-4]` cuts away ` sat` or something? Looks scary. Should we maybe use `.split()` for this?

Reply:
Positive, [0:-4] cuts away msat suffix. Python is not my main language, all the python that I learned was in my LNBITS PR's haha. I'm using this code in my own fork since a month ago and monitoring the logs. If you look at the code of the spark integration, [0:-4] is already being used to remove this suffix from the some objects fields returned by CLN.